s3transfer/futures.py

```diff
@@ -78,8 +78,9 @@ def cancel(self):
 
 class TransferMeta(object):
     """Holds metadata about the TransferFuture"""
-    def __init__(self, call_args=None):
+    def __init__(self, call_args=None, id=None):
```

Comment:
It should not be an issue because it is isolated to the class but I can update it to something more explicit.